runtime/parser/keyword.go

```diff
@@ -19,47 +19,116 @@
 package parser
 
 const (
-	keywordIf          = "if"
```

Comment:
It doesn't seem like these need to be accessible outside of the `parser` module; I'd lowercase prefix all these (as well as the names of the keywords maps) so we don't expose more implementation details than is strictly necessary.